server/app/views/applicant/ProgramIndexView.java

```diff
@@ -286,41 +286,37 @@ private LiTag programCard(
               preferredLocale, cardData.latestSubmittedApplicationStatus().get()));
     }
     programData.with(title, description);
-
-    // Add info link.
-    String infoUrl =
-        controllers.applicant.routes.ApplicantProgramsController.view(applicantId, program.id())
-            .url();
-    ATag infoLink =
-        new LinkElement()
-            .setId(baseId + "-info-link")
-            .setStyles("mb-2", "text-sm", "underline")
-            .setText(messages.at(MessageKey.LINK_PROGRAM_DETAILS.getKeyName()))
-            .setHref(infoUrl)
-            .asAnchorText()
-            .attr(
-                "aria-label",
-                messages.at(
-                    MessageKey.LINK_PROGRAM_DETAILS_SR.getKeyName(),
-                    program.localizedName().getOrDefault(preferredLocale)));
-    programData.with(div(infoLink));
-
-    // Add external link if it is set.
+    // Add external link if it is set else info link.
     if (!program.externalLink().isEmpty()) {
       ATag externalLink =
           new LinkElement()
               .setId(baseId + "-external-link")
               .setStyles("mb-2", "text-sm", "underline")
-              .setText(messages.at(MessageKey.EXTERNAL_LINK.getKeyName()))
+              .setText(messages.at(MessageKey.LINK_PROGRAM_DETAILS.getKeyName()))
               .setHref(program.externalLink())
               .opensInNewTab()
```

Comment:
**suggestion:** Seems a little funny to me that this would open in a new tab and the other wouldn't, since now they have the same text. Thoughts on using the conditional to set the href value, but have the externalLink definition be the same in either case to ensure the same look and functionality?

Reply:
Actually it is a good call to use a conditional statement outside to assign the href and use the same formatting. I had thought about it but later forgot. Have made the changes and used the most of programDetails formatting and added .opensInNewTab() and .SetIcon() from externalLink..

Reply:
Perfect, thanks!